Shared/SwiftAttributes.swift

```diff
@@ -0,0 +1,12 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+public var lessThaniOS13: Bool {
```

Comment:
Used only in one place, can we make it a lambda/location function for that one scenario instead of a global variable?